src/helpers/MeasureBundleHelpers.ts

```diff
@@ -122,6 +122,61 @@ export function hasMultipleIPPs(group: fhir4.MeasureGroup) {
   );
 }
 
+/* 
+  Finds the measure observation that reference the desired population in their criteria reference. If one exists,
+  sets the result to false and the observations to null
+*/
+export function nullCriteriaRefMeasureObs(
+  group: fhir4.MeasureGroup,
+  populationResults: PopulationResult[],
+  desiredPopulationType: PopulationType
+): void {
+  const popResults = populationResults.filter(result => result.populationType === PopulationType.OBSERV);
+  if (desiredPopulationType === PopulationType.NUMER && popResults.length === 1) {
+    // If only one measure-observation population, we know it relates to numerator
+    popResults[0].result = false;
+    popResults[0].observations = null;
+  } else {
+    const measureObs = getCriteriaRefMeasureObs(group, popResults, desiredPopulationType);
+    if (measureObs) {
+      measureObs.result = false;
+      measureObs.observations = null;
+    }
+  }
+}
+
+/*
+   Finds the measure observation that references the desired population in its criteria reference 
+   and returns its populationResult
+*/
+export function getCriteriaRefMeasureObs(
```

Comment:
I wonder if we can unify this function with the `getRelevantIPPFromPopulation` since they're very very similar

Reply:
Yeah.. good point. Though the functions do serve slightly different purposes. One is identifying a `fhir4.MeasurePopulation` within a group that is referenced via the criteriaReference extension. The other one is finding a `PopulationResult` for a measure observation that `references` a certain population.

I'll go ahead and update the implementation and the naming of this function to clearly reflect that, and will be sure to re-use existing helper functions where appropriate

Reply:
I also think that, to be more explicit, I'll go ahead and update it to return null instead of undefined.